datasets/amazon-books/README.md

```diff
@@ -0,0 +1,23 @@
+<h1 align="center"> :bar_chart: Amazon book ratings stream :bar_chart:</h1>
+
+## :speech_balloon: About
+
+Amazon started as a music, video and book web store. During the recent period it
+grew to technology giant that sells varius different products and services. 
+Todaty, Amazon e-commerce platform is one of the biggest in the world. According to 
+Amazon data and at the time of writing, they recive 1.6 milion orders daily. It 
+makes sense to analyze thair e-commerce data because of scale.   
+
+## :open_file_folder: Dataset
+
+Books played a vital role in Amazon's business scaling and a large part of daily sales
+are book sales. So it makes sense to use it as a streaming dataset. 
+Dataset consists of user book ratings. The ratings were aquired from: [Amazon dataset](https://jmcauley.ucsd.edu/data/amazon)
+
+## :fast_forward: Streaming book ratings
+
+Check instructions in root [README](../../README.md) how to start data stream. 
+
```

Comment:
Please run this through Grammarly.

Reply:
The best part is that I actually did it, but I always correct mistakes in grammarly on the web and not in repo 😅.